library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/GroupResourcePack.java

```diff
@@ -77,7 +77,7 @@ public GroupResourcePack(@NotNull ResourceType type, @NotNull List<? extends Res
 	 * @return the list of the matching resource packs
 	 */
 	public @UnmodifiableView List<? extends ResourcePack> getPacks(String namespace) {
-		return Collections.unmodifiableList(this.namespacedPacks.get(namespace));
+		return Collections.unmodifiableList(this.namespacedPacks.getOrDefault(namespace, Collections.emptyList()));
```

Comment:
When merging (or before merge) this probably should be split to use an if instead (get and store into a local, if not null then return the unmodifiable call, else return emptyList()).